docs/src/installation.md

```diff
@@ -8,6 +8,28 @@ Pkg.add("RCall")
 
 ## Customizing the R installation
 
+There are two ways to configure the R installation used by RCall.jl:
+
+  * [Using Julia's Preferences system](#Customizing-the-R-installation-using-Julia's-Preferences-system)
+  * [At install time, using the `R_HOME` environment variable](#Customizing-the-R-installation-at-install-time)
```

Comment:
nit: this is at _build_ time for RCall. You can rebuild a package that's already been installed. (IIRC TimeZones also takes advantage of this to make its data are up to date.)

Reply:
I suppose I was trying to contrast the two from the user's perspective, so that I could use consistent shorthand on this page. In a sense talking about build-time instead would also be misleading, since in terms of being a correct description of the implementation, R_HOME and the Preferences approach are *both* configured at build-time, but Preferences has the advantage that it triggers a build at the correct time and keeps multiple builds around. I have switched to using the `R_HOME` environment variable as a shorthand.